activerecord/lib/active_record/railtie.rb

```diff
@@ -26,6 +26,7 @@ class Railtie < Rails::Railtie # :nodoc:
 
     config.active_record.use_schema_cache_dump = true
     config.active_record.maintain_test_schema = true
+    config.active_record.belongs_to_required_by_default = true
```

Comment:
Each values will be shared in initializers. It depends on order.

```
# config/initializers/a.rb
Rails.application.config.active_record.belongs_to_required_by_default = false                                                                         
Rails.application.config.active_record.maintain_test_schema = false

# config/initializers/b.rb
puts Rails.application.config.active_record.belongs_to_required_by_default #=> false
puts Rails.application.config.active_record.maintain_test_schema #=> false
```

And `initializer "active_record.set_configs"` sends values to `ActiveRecord::Base`, so I thought it was ok after initialize.